torch/_dynamo/variables/torch.py

```diff
@@ -476,6 +476,22 @@ def get_state_from_generator():
                 **options,
             )
 
+            def track_out_tensors(new_tensor_variable, old_tensor_variable):
+                local_name = tx.find_symbolic_locals_name(old_tensor_variable)
+                if local_name:
+                    tx.symbolic_locals[local_name] = new_tensor_variable
```

Comment:
This is a preexisting condition, but I'm wondering if the original logic here was even correct. `out=` arguments are mutated, and so we are obligated to update the variable trackers for ALL aliases to the same tensor. It's not clear to me that only updating symbolic locals here is enough. If the logic here is indeed wrong, we can probably induce a failure by `a = b` and then doing an `out=` operation on a/b.